project/Dependencies.scala

```diff
@@ -19,8 +19,8 @@ object Dependencies {
   val awsSQS = "software.amazon.awssdk" % "sqs" % awsClientV2Version
   val awsCloudWatch = "com.amazonaws" % "aws-java-sdk-cloudwatch" % awsClientVersion
   val scalaz = "org.scalaz" %% "scalaz-core" % "7.3.7"
-  val kinesis = "com.gu" % "kinesis-logback-appender" % "2.0.3"
-  val logstash = "net.logstash.logback" % "logstash-logback-encoder" % "4.9"
+  val kinesis = "com.gu" % "kinesis-logback-appender" % "2.1.1"
+  val logstash = "net.logstash.logback" % "logstash-logback-encoder" % "7.3"
```

Comment:
logstash-logback-encoder 7.4 available, but I think it requires Java 17+ (?) not 100% sure however

Reply:
good spot, I just went with what intellij auto suggested.  Docs say it (only) needs java 8 actually https://github.com/logfellow/logstash-logback-encoder#java-version-requirements

Reply:
hm ok, we didn't upgrade in our docs project because we got an error. Maybe it pulls in an newer jackson version than Play ships with, can't remember anymor.